### DISCUSSION TITLE
Livewire encountered corrupt data when trying to hydrate the component

### DISCUSSION THREAD

**Discussion by mohsenbostan** [livewire/livewire#4226] [1. Help]
Hi,
Livewire components work fine on my local machine without any errors, but not on my production server.

Here is one of my errors when using livewire actions (using wire:model, button clicks, etc. ):
![image](https://user-images.githubusercontent.com/20592417/142138366-b4dae24c-b509-40e5-8a84-57b330488927.png)

The other occurs with wire:model:
![image](https://user-images.githubusercontent.com/20592417/142138600-679fe4d3-5d1d-4547-95fe-fd1be66a99ba.png)

These are all of my properties, and I modify the values just with wire:model.
```
    public $username = '';
    public $is_verified = 0;

    protected $rules = [
        'username' => 'required|min:3|unique:users,username',
        'is_verified' => 'required',
    ];
```

**Comment by joshhanley** [comment]
@mohsenbostan I think I've seen that before, I believe it might be assigning the empty string to username.

Any chance you could set that to null and test it to see if it changes anything?

**Reply by mohsenbostan** [reply]
I think there was something wrong with Nginx because I've solved the problem by adding localhost to server_name. So that's not the problem. If that would cause any problems, then I should have received the same error on my machine as well.
